wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceDetailsViewActionsHandler.swift

```diff
@@ -142,13 +150,18 @@ final class DeviceDetailsViewActionsHandler: DeviceDetailsViewActions, Observabl
 }
 
 extension DeviceDetailsViewActionsHandler: ClientRemovalObserverDelegate {
+
+    @MainActor
     func present(
         _ clientRemovalObserver: ClientRemovalObserver,
         viewControllerToPresent: UIViewController
     ) {
-        UIViewController.presentTopmost(viewController: viewControllerToPresent)
+        if !(UIApplication.shared.topmostViewController()?.presentedViewController is UIAlertController) {
+            UIViewController.presentTopmost(viewController: viewControllerToPresent)
+        }
     }
```

Comment:
It would be easier to read:
```swift
let presentsAlert = UIApplication.shared.topmostViewController()?.presentedViewController is UIAlertController
if presentsAlert { return }
UIViewController.presentTopmost(viewController: viewControllerToPresent)
```

Reply:
I think we could keep the concise version as is here. IMHO it is quite straight forward